packages/jest-cli/src/TestRunner.js

```diff
@@ -16,9 +16,14 @@ import type {
 import type {GlobalConfig, ReporterConfig} from 'types/Config';
 import type {Context} from 'types/Context';
 import type {Reporter, Test} from 'types/TestRunner';
-import type {PathPattern} from './SearchSource';
+import type {PathPatternObject} from './SearchSource';
 
 import {formatExecError} from 'jest-message-util';
+import {
+  addResult,
+  buildFailureTestResult,
+  makeEmptyTestResult,
+} from './test_result_utils';
```

Comment:
it can't be `makeEmptyAggregatedResult` cause it has other utility functions in it (make test result, add test to aggregated result).
I'll rename it to helpers, but to me it's just another word for utilities :)